packages/svelte/tests/validator/samples/ignore-warning-dash-runes/warnings.json

```diff
@@ -0,0 +1,26 @@
+[
+	{
+		"code": "a11y_missing_attribute",
+		"end": {
+			"column": 29,
+			"line": 5
+		},
+		"message": "`<img>` element should have an alt attribute (this warning was tried to silence using code a11y-missing-attribute. In runes mode, use the new code a11y_missing_attribute instead)",
```

Comment:
I think it would be simpler and more helpful to warn on `<!-- svelte-ignore a11y-missing-attribute -->` in runes mode, and leave the other warning untouched

Reply:
(working on this)